src/scimlfunctions.jl

```diff
@@ -1901,6 +1959,22 @@ function SDEFunction{iip,true}(f,g;
                    _colorvec = colorvec
                  end
 
+                 giip = isinplace(g,4,"g")
```

Comment:
@ChrisRackauckas does this work if a user gives both iip and oop versions but sets `iip=false? That is what ModelingToolkit does by default.